File: app/src/main/java/com/example/amit/uniconnexample/Notificationmodel.java
package com.example.amit.uniconnexample;

/**
 * Created by amit on 21/1/17.
 */

public class Notificationmodel {
    String img,txt,key,post_key;
    public Notificationmodel(String img,String txt,String key,String post_key){
        this.img=img;
        this.txt=txt;
        this.key=key;
        this.post_key=post_key;
    }

    public Notificationmodel() {
    }

    public String getImg() {
        return img;
    }

    public String getTxt() {
        return txt;
    }

    public String getKey() {
        return key;
    }

    public String getPost_key() {
        return post_key;
    }
}

File: app/src/main/java/com/example/amit/uniconnexample/Model/Notificationmodel.kt
package com.example.amit.uniconnexample

/**
 * Created by amit on 21/1/17.
 */

class Notificationmodel {
    var img: String? = null
        internal set
    var txt: String? = null
        internal set
    var key: String? = null
        internal set
    var post_key: String? = null
        internal set

    constructor(img: String, txt: String, key: String, post_key: String) {
        this.img = img
        this.txt = txt
        this.key = key
        this.post_key = post_key
    }

    constructor() {}
}
